src/libs/WorkflowUtils.ts

```diff
@@ -136,14 +160,7 @@ function convertPolicyEmployeesToApprovalWorkflows({employees, defaultApprover,
         });
     }
 
-    // Add a flag to each approver to indicate if they are in multiple workflows
-    return sortedApprovalWorkflows.map((workflow) => ({
-        ...workflow,
-        approvers: workflow.approvers.map((approver) => ({
-            ...approver,
-            isInMultipleWorkflows: approverCountsByEmail[approver.email] > 1,
-        })),
-    }));
+    return {approvalWorkflows: sortedApprovalWorkflows, usedApproverEmails: [...usedApproverEmails], availableMembers: sortedApprovalWorkflows.at(0)?.members ?? []};
```

Comment:
And then, we don't need to spread again here

```suggestion
    return {approvalWorkflows: sortedApprovalWorkflows, usedApproverEmails, availableMembers: sortedApprovalWorkflows.at(0)?.members ?? []};
```